src/test/kotlin/no/nav/syfo/dialogmote/api/v2/GetDialogmoteApiV2Spek.kt

```diff
@@ -107,6 +109,46 @@ class GetDialogmoteApiV2Spek : Spek({
                         }
                     }
 
+                    it("should return DialogmoteList based on VeilederIdent") {
```

Comment:
Savner test av tilgangstyring her. Evt. også at det kun kommer ut dialogmoter med riktig status.
Kan se `GetDialogmoteEnhetApiV2Spek` hvor disse tingene er gjort, bare for enhet og ikke veilederident.